code/game/machinery/syndicatebeacon.dm

```diff
@@ -117,20 +117,17 @@
 	if(user)
 		to_chat(user, "<span class='notice'>You deactivate the beacon.</span>")
 
-
-/obj/machinery/power/singularity_beacon/attack_ai(mob/user as mob)
+/obj/machinery/power/singularity_beacon/emergency/attack_ai(mob/user)
 	return
```

Comment:
Why is this here if it just gets overridden below?

Reply:
whoops, this should not be on the beacon subtype